MICADO/docs/example_notebooks/MICADO_4_stars.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c91d6ba9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from astropy import units as u\n",
+    "\n",
+    "import scopesim_templates as sim_tp\n",
+    "import scopesim as sim"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "04224d82",
+   "metadata": {},
+   "source": [
+    "# Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e962001c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Gijs's local computer\n",
+    "# sim.download_packages([\"Armazones\", \"ELT\", \"MAORY\", \"MICADO\"])\n",
+    "\n",
+    "# Kieran's computer -> redirects ScopeSim to folder containing IRDB \n",
+    "# MicadoWise could do this too and clone daily from github to save running time\n",
+    "sim.rc.__config__[\"!SIM.file.local_packages_path\"] = \"F:\\Work\\irdb\""
```

Comment:
The inst_pkgs symlink made it not necessary to specify a path, because it would just use the local checkout, so you would always run the notebook with the version of the irdb that the notebook was in, thereby preventing having paths to peoples local computer in the notebook.

Not sure how this works on Windows, but windows supports symlinks as well, so I guess it should work.